web/src/App.js

```diff
@@ -0,0 +1,33 @@
+import { FatalErrorBoundary } from '@redwoodjs/web'
+import { RedwoodApolloProvider } from '@redwoodjs/web/apollo'
+import FatalErrorPage from 'src/pages/FatalErrorPage'
+import supertokens from "supertokens-auth-react";
+import sessions from "supertokens-auth-react/recipe/session";
+import emailpassword from "supertokens-auth-react/recipe/emailpassword";
+
+import Routes from 'src/Routes'
+
+import './index.css'
+
+supertokens.init({
+  appInfo: {
+    apiDomain: "http://localhost:8910/",
+    appName: "SuperTokens RedwoodJS",
+    websiteDomain: "http://localhost:8910/",
+    apiBasePath: "/.netlify/functions/auth"
+  },
+  recipeList: [
+    sessions.init(),
+    emailpassword.init()
+  ]
+});
```

Comment:
User needs to call supertokens.init on the frontend to initialise the frontend logic / UI. The reason this is here is that the `init` function needs to have been called before `supertokens.canHandleRoute()` is called (see Routes.js file)